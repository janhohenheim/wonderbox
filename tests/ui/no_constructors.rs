use wonderbox::autoresolvable;

struct List;

#[autoresolvable]
impl List {
}

fn main() {}
