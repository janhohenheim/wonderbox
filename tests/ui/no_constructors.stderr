error: Expected one constructor, found 0
 --> $DIR/no_constructors.rs:6:1
  |
6 | impl List {
  | ^^^^
